src/spring-cloud/azext_spring_cloud/vendored_sdks/appplatform/_configuration.py

```diff
@@ -36,7 +36,7 @@ def __init__(
         if subscription_id is None:
             raise ValueError("Parameter 'subscription_id' must not be None.")
         if not base_url:
-            base_url = 'https://management.azure.com'
+            base_url = 'https://eastus2euap.management.azure.com'
```

Comment:
This looks like for testing. Please confirm.

Reply:
> This looks like for testing. Please confirm.

updated.